.github/workflows/integration.yml

```diff
@@ -20,4 +20,5 @@ jobs:
     - name: Install OVS
       run: sudo apt-get install openvswitch-switch
     - name: Run integration tests
-      run: sudo make test-integration
+      # Use the specified go version instead the default one
+      run: sudo make GOROOT="$(go env GOROOT)" GOPATH="$(go env GOPATH)" GO="$(go env GOROOT)/bin/go" test-integration
```

Comment:
interesting, I would have expected that only `GO="$(go env GOROOT)/bin/go"` was required, as long as our Makefile used `$(GO)` consistently (and not `go`).

Reply:
If we don't set these env var, the 1.12 toolchain in go path will break the build.